File: challenge_problem_1/src/pickup_object/src/pickup_object.cpp
#include <ros/ros.h>

int main(int argc, char** argv)
{
  //init the ROS node
  ros::init(argc, argv, "pickup_object");

  ros::NodeHandle nh;

  //sleep for a moment while everything starts up
  ros::Duration(1.0).sleep();

  // your code goes here:



  ROS_INFO("Finished Picking up the cup!");

  return 0;
}

File: challenge_problem_1/src/pickup_object/scripts/pickup_object.py
#!/usr/bin/env python

import rospy


if __name__ == '__main__':

    #initialize the node
    rospy.init_node('pickup_object', anonymous=True)

    #sleep for a moment while everything starts up
    rospy.sleep(1.0)

    # your code here:
    rospy.loginfo("Finished Picking up the cup!")